### PR TITLE
Preseed wireshark debconf question

### PR DESCRIPTION
**Pull request by tmochida**

### CODE REVIEW

File: modules/ocf_desktop/manifests/wireshark.pp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class ocf_desktop::wireshark {
+  package {
+    'wireshark-common':
+      responsefile => '/var/cache/debconf/wireshark-common.preseed',
+      require      => File['/var/cache/debconf/wireshark-common.preseed'];
+
+    'wireshark':
+      ensure => present;
```
Comment (chriskuehl) [inline modules/ocf_desktop/manifests/wireshark.pp]: if you want you can just do `'wireshark':;` since `ensure => present` is default

**Comment by chriskuehl** [comment]
thanks for doing this! lgtm

**Comment by tmochida** [comment]
Updated PR per suggestion from @chriskuehl

**Comment by chriskuehl** [comment]
sheep it

**Pull request merged**